CONTRIBUTING.md

```diff
@@ -22,33 +24,98 @@ Before contributing we ask that you review the following guidelines. If you thin
 
 [Styleguides](#styleguides)
 
-- [Pull Request Titles](#pull-request-titles)
+- [Pull Request Titles Styleguide](#pull-request-titles-styleguide)
+- [Commits Styleguide](#commits-styleguide)
 - [Javascript Styleguide](#javascript-styleguide)
 - [Specs Styleguide](#specs-styleguide)
 - [Documentation Styleguide](#documentation-styleguide)
 
-[Additional Resources](#additional-resources)
-
 ## Code Of Conduct
 
 This project and everyone participating in it is governed by the [Nacelle Code of Conduct](CODE_OF_CONDUCT.md). All participants are expected to abide by this code. Please report unacceptable behavior to [support@getnacelle.com](mailto:support@getnacelle.com).
 
 ## Getting Started
 
-Nacelle-js is a set of tools designed to empower developers to build unique, performant & scalable storefronts. The repository is broken the following parts.
+### Setting up your development environment
+
+This project uses [Lerna](https://lerna.js.org) to coordinate dependencies throughout the monorepo. Lerna allows us to make changes to a package and see the effects of those changes in other packages and example projects, without the need to manually mangage package symlinks via `npm link`.
```

Comment:
Hey Nick! I think there is a little typo on "manually manage" here

Reply:
LOLOL @ "mangage" - good find, @KrisQ!

Fixed in https://github.com/getnacelle/nacelle-js/pull/36/commits/9dd0afede48210412057c4e5601d93a1e7b7235f.